### ISSUE TITLE
About performing a transfer learning task using FD data

### ISSUE THREAD

**Issue by start0314** [emadeldeen24/TS-TCC#35]
Thank you very much for sharing such an impressive work.

I would like to ask about the procedure for reproducing the transfer learning task presented in the paper.

Using main.py, I pre-trained on FD-A and then changed the training mode to fine-tune to perform the transfer learning task on FD-B data.

In all cases of A->B, A->C, and A->D, the accuracy is coming out as 1.0.

Is there anything I might have missed in reproducing the transfer learning task?

Thank you in advance.

**Comment by emadeldeen24** [comment]
For scenario A-> B, we pretrain on A, fine-tune with A, then directly test on B.
